core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -146,7 +146,12 @@ class Controller(val instance: ControllerInstanceId,
         complete {
           loadBalancer
             .invokerHealth()
-            .map(_.map(i => s"invoker${i.id.toInt}" -> i.status.asString).toMap.toJson.asJsObject)
+            .map(_.map(i => {
+              i.id.hostname
+                .map(hostname => s"invoker${i.id.toInt}/$hostname" -> i.status.asString)
+                .orElse(Some(s"invoker${i.id.toInt}" -> i.status.asString))
+                .get
```

Comment:
`get` is unsafe and can throw exceptions. In this case you can statically guarantee no exceptions by using `getOrElse` like:

```scala
_.map(i => {
  val key = i.id.name.map(name => s"invoker${i.id.toInt}/$name").getOrElse(invoker${i.id.toInt})
  key -> i.status.asString
}).toMap.toJson.asJsObject
```